packages/bueno/project.json

```diff
@@ -1 +1,109 @@
-{}
+{
+  "namedInputs": {
+    "tests": ["{projectRoot}/**/*.test.ts"],
+    "source": [
+      "{projectRoot}/esbuild.mjs",
+      "{workspaceRoot}/tsconfig.json",
+      "{projectRoot}/tsconfig.json",
+      "{projectRoot}/src/**/*([^.]).!(test.ts)"
+    ]
+  },
+  "targets": {
+    "clean": {
+      "dependsOn": [],
+      "inputs": [],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "rimraf dist",
+        "cwd": "packages/bueno"
+      }
+    },
+    "cached:build:esbuild": {
+      "dependsOn": ["^build"],
+      "inputs": ["^source", "source"],
+      "outputs": ["{projectRoot}/dist"],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "node esbuild.mjs",
+        "cwd": "packages/bueno"
+      }
+    },
+    "cached:build:definitions": {
+      "dependsOn": ["cached:build:esbuild"],
+      "inputs": ["^source", "source"],
+      "outputs": ["{projectRoot}/dist/definitions"],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "tsc -d --emitDeclarationOnly --declarationDir dist/definitions",
+        "cwd": "packages/bueno"
+      }
+    },
+    "build": {
+      "dependsOn": ["cached:build:esbuild", "cached:build:definitions"],
+      "inputs": [],
+      "outputs": [],
+      "executor": "nx:noop"
+    },
+    "dev:esbuild": {
+      "dependsOn": [],
+      "inputs": [],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "node esbuild.mjs dev",
+        "cwd": "packages/bueno"
+      }
+    },
+    "dev:definitions": {
+      "dependsOn": [],
+      "inputs": [],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "tsc -d --emitDeclarationOnly --declarationDir dist/definitions -w",
+        "cwd": "packages/bueno"
+      }
+    },
+    "dev": {
+      "dependsOn": [],
+      "inputs": [],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
+          "nx run bueno:\"dev:esbuild\"",
+          "nx run bueno:\"dev:definitions\""
+        ],
```

Comment:
Normally, we should **never** run multiple `nx run` in parallel, since they both might automatically run the same dependent tasks at the same time. I made an exception here for `dev` since both tasks have no dependencies and since I just want it to work the same as in `master`.